.github/actions/setup/action.yml

```diff
@@ -0,0 +1,35 @@
+name: "Setup & build"
+description: "Setup node, install dependencies & build"
+runs:
+  using: composite
+  steps:
+    - uses: actions/setup-node@v2
+      with:
+        node-version-file: '.nvmrc'
+    - uses: actions/cache@v2
+      id: build-cache
+      with:
+        path: |
+          packages/bueno/dist
+          packages/headless/dist
+          packages/auth/dist
+          packages/atomic/dist
+          packages/atomic/docs
+          packages/atomic/headless
+          packages/atomic/loader
+          packages/atomic/storybook-static
+          packages/atomic/www
+        key: ${{ github.sha }}
```

Comment:
Correct.

I addressed this [above](https://github.com/coveo/ui-kit/pull/1567#discussion_r764144387), but it looks like we wrote both of our comments at once. This may be over-optimizing, since builds are gonna become faster with esbuild.